hpx/runtime/actions/continuation_impl.hpp

```diff
@@ -54,11 +55,10 @@ namespace hpx { namespace actions {
         template <typename Archive>
         HPX_FORCEINLINE void serialize(Archive& ar, unsigned int const)
         {
-            ar & cont_ & target_;
+            ar& cont_& target_;
```

Comment:
I think this formatting was on purpose cause it's the & operator right?

Reply:
clang-format always messes this up. We either live with it or surround this kind of code with `// clang-format off/on` comments.

Reply:
I can manually revert the clang-formatting changes on these files.

Reply:
Yes please, if it's not too much effort. They will need the `// clang-format` comments though.

Reply:
I have manually reverted clang-formatting changes on these files.

Reply:
Thank you!